processor/src/test/java/com/linecorp/decaton/processor/runtime/ProcessorSubscriptionTest.java

```diff
@@ -46,35 +42,21 @@
 
 public class ProcessorSubscriptionTest {
```

Comment:
While implementing state transition integration test, I noticed that `ConsumerRebalanceListener#onPartitionsRevoked` is no longer triggered when consumer startup. (i.e. transition to `RUNNING` from `INITIALIZING` directly).

- Fix also this test not to call `onPartitionsRevoked`
- Besides, I came up with it's better to simulate rebalance listener invocation by injecting pollTask in each test [like this](https://github.com/line/decaton/pull/48/files#diff-633ecc9f820b590d094c66ac75f62ef8R90) instead of trying to simulate by mock consumer impl